src/main/dynamo/Beyonce.ts

```diff
@@ -30,6 +30,7 @@ export class Beyonce {
 
     if (options.jayz !== undefined) {
       this.jayz = options.jayz
+      table.getEncryptionBlacklist().forEach(this.jayz.encryptionBlacklist.add)
```

Comment:
Hmm it looks like this shouldn't compile - does `npm run build` work for you?

Asking since JayZ doesn't have an encryptionBlacklist property on it: 

```TypeScript
export declare class JayZ {
    private keyProvider;
    private encryptor;
    private maxUsesPerDataKey;
    private timesDataKeyUsed;
    private dataKey?;
    constructor(config: JayZConfig);
    encryptItem<T, K extends keyof T>(item: T, fieldsToEncrypt: K[]): Promise<EncryptedJayZItem<T, K>>;
    decryptItem<T, K extends keyof T>(encryptedJayZItem: EncryptedJayZItem<T, K>): Promise<T>;
    private getDataKey;
}

```

And it looks like we already populate `Table`'s encryption blacklist - so I don't think we'd need this line here right?

Reply:
Yes, I just realized by local master branch was outdated. I have removed this bit.